packages/server/src/db/linkedRows/LinkController.ts

```diff
@@ -221,9 +221,15 @@ class LinkController {
                 link.id !== row._id && link.fieldName === linkedSchema.name
             )
 
+            // check all the related rows exist
+            const foundRecords = await this._db.getMultiple(
+              links.map(l => l.id),
+              { allowMissing: true, excludeDocs: true }
+            )
```

Comment:
I'm confused, this doesn't seem to line up with the PR description. It looks like you're checking for existing links to a document on save, not looking for links to documents that don't exist?

Reply:
we're deciding whether or not the validation is correct by getting a list of rows, the way the system works it retrieves a list of link documents, this then makes sure those link documents reference something valid.